packages/monarch_grpc/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: monarch_grpc
 description: gRPC interfaces for Monarch. Monarch is a tool for building Flutter widgets in isolation. It makes it easy to build, test and debug complex UIs.
-version: 2.2.0
+version: 2.3.0
```

Comment:
I don't see this new version referenced from other monarch modules, like the controller or preview_api.

Reply:
done, forgot to update